packages/apollo-server/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/apollographql/apollo-server#readme",
   "dependencies": {
+    "@types/express": "4.17.13",
```

Comment:
Does it need to be pinned? ):

Reply:
It's pinned like this in `apollo-server-express` so I figured doing the same thing was reasonable.

We've had a lot of challenges with the Express DefinitelyTyped packages before.  They factored out a huge part of the types from `@types/express` into `@types/express-serve-static-core` but the dependency of the former on the latter doesn't declare a specific version, so it's easy to accidentally install a pair of versions that don't work together.  So unfortunately trying to pin more precisely does seem helpful.  If they get behind we're happy to update everything...